CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## Next release
+## Internal
+- Add remote toggle for screen change animations
+- Add instrumentation around Flow events
+
```

Comment:
Can you please add the consent text changelog too?

Reply:
Done in the `Feature` section.